contents/handbook/engineering/beginners-guide/developer-workflow.md

```diff
@@ -23,7 +23,7 @@ you are free to use whatever IDE makes the most sense to you.
     - Use [print()](https://realpython.com/python-print/) as needed for debugging
     - Some developers prefer [Pycharm](https://www.jetbrains.com/pycharm/) for local development
 8. Run backend tests
-    - `bin/tests posthog` runs only posthog tests excluding ee tests
+    - `bin/tests posthog` runs only posthog tests excluding PostHog Scale tests
     - `./bin/tests ee/clickhouse/queries/test/test_trends.py -k test_active_user_math` for specific tests
 9. Commit changes to git branch
 10. Open PR for review
```

Comment:
So the directory this is referring to is actually called `ee` (https://github.com/PostHog/posthog/tree/master/ee)

Given this is more of a technical doc, do you think we may want to say something more like "runs only posthog tests excluding PostHog Scale (ee directory) tests" or "runs only posthog tests excluding ee (PostHog Scale) tests"?

Reply:
Thanks, @kpthatsme. This got me thinking a bit more about the context so I've tried to make it very clean in https://github.com/PostHog/posthog.com/pull/1847/commits/1234a0e4adbab50c79db0e5b34b9af65b3c24e41